tests/Unit/Contrib/AgentExporterTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Contrib\Unit;
+
+use OpenTelemetry\Contrib\Jaeger\AgentExporter;
+use OpenTelemetry\SDK\Trace\SpanExporterInterface;
+use OpenTelemetry\Tests\Unit\SDK\Util\SpanData;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\Contrib\Jaeger\AgentExporter
+ * @covers OpenTelemetry\Contrib\Jaeger\JaegerTransport
+ * @covers OpenTelemetry\Contrib\Jaeger\ThriftUdpTransport
+ */
+class AgentExporterTest extends TestCase
+{
+    public function test_happy_path()
+    {
+        $exporter = AgentExporter::fromConnectionString(
+            'http://127.0.0.1:80', //This isn't realistic I imagine
```

Comment:
that's fine for a unit test, and you can make it more obviously fake like some other tests do... `localhost:1234` etc

Reply:
Ah forgot about those comments! (removed them now) I think I looked super briefly to try and figure out what they're supposed to be in reality but couldn't find anything quickly and gave up.

Might be something to loop back on post functional testing!